stable/datadog/templates/daemonset.yaml

```
@@ -274,6 +284,16 @@ spec:
             path: {{ default "/var/lib/datadog-agent/logs" .Values.datadog.logsPointerHostPath | quote }}
           name: pointerdir
         {{- end }}
+        {{- if (and (.Values.datadog.logsEnabled) (.Values.datadog.logsConfigContainerCollectAll)) }}
+        - hostPath:
+            path: /var/log/pods
+          name: logpodpath
+        {{- end }}
+        {{- if (and (.Values.datadog.containerLogsPath) (not (eq .Values.datadog.containerLogsPath ""))) }}
```

the following line should be enough

```suggestion
        {{- if .Values.datadog.containerLogsPath }}
```